### PR TITLE
Add most important notes to readme

### PR DESCRIPTION
**Pull request by fschindler**
@okwast @ckru-cbb feel free to add some more of your special knowledge!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,64 @@
...
+```
+cd backend
+hasura console --admin-secret myadminsecretkey
+```
+
+And visit http://localhost:9695
+
+### Apply migrations, seeds and metadata
+
+Go and get `exportedseed.sql` (ask somebody) and copy it to `./backend/seeds/default`.
```
Comment (fschindler) [inline README.md:28]: @ckru-cbb what is the best way to get the current version(s) of this file to every developer machine?

File: README.md [hunk 2]
```diff
@@ -1,9 +1,64 @@
...
+```
+cd backend
+hasura migrate apply --admin-secret myadminsecretkey
+hasura seeds apply --admin-secret myadminsecretkey
+hasura metadata apply --admin-secret myadminsecretkey
+```
+
+## Keycloak
+
+Go and get `staging-realm.json` (ask sombody) and copy the file to `.keycloak/`.
```
Comment (fschindler) [inline README.md:39]: @okwast you've send a version of this file to me... which one should we use? Should we commit a new one?

Comment (okwast) [reply README.md:39]: No, we should keep the one in the repository. It seems like I missed that it was added to the repo.

On the other hand do you think it is ok to have this file in the repo or is it too specific?

Comment (fschindler) [reply README.md:39]: IMHO we should keep this file in the repo for the moment. E.g. the role names in this file have to match some metadata files in the `./backend` directory.

For the long run we should revisit this file and check if we can create a more generic version.

Comment (okwast) [reply README.md:39]: Perfect. As this is your discussion, feel free to resolve it.

**Pull request closed without merging**